.github/workflows/main-image.yml

```diff
@@ -21,26 +25,50 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
-      - name: Build Image
-        id: build-image
+      - name: Install qemu dependency
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y qemu-user-static
+
+      - name: Build image linux/amd64
+        id: build-image-amd64
         uses: redhat-actions/buildah-build@v2
         with:
           image: operator-pipelines-images
-          tags: latest ${{ github.sha }}
+          tags: linux-amd64
+          arch: amd64
           dockerfiles: |
             ./operator-pipeline-images/Dockerfile
 
-      - name: Push To quay.io
-        id: push-to-quay
-        uses: redhat-actions/push-to-registry@v2
+      - name: Build image linux/ppc64le
+        id: build-image-ppc64le
+        uses: redhat-actions/buildah-build@v2
         with:
-          image: ${{ steps.build-image.outputs.image }}
-          tags: ${{ steps.build-image.outputs.tags }}
-          registry: quay.io/redhat-isv
+          image: operator-pipelines-images
+          tags: linux-ppc64le
+          arch: ppc64le
+          dockerfiles: |
+            ./operator-pipeline-images/Dockerfile-ppc64le
+
+      # Authenticate to container image registry to push the image
+      - name: Podman Login to quay.io
+        uses: redhat-actions/podman-login@v1
+        with:
+          registry: quay.io
           username: ${{ secrets.REGISTRY_USERNAME }}
           password: ${{ secrets.REGISTRY_PASSWORD }}
         if: ${{ github.event_name == 'push' }}
 
+      - name: Create and add to manifest
+        run: |
+          buildah manifest create ${{ env.IMAGE_NAME }}
+          buildah manifest add ${{ env.IMAGE_NAME }} ${{ env.IMAGE_NAME }}:linux-amd64
+          buildah manifest add ${{ env.IMAGE_NAME }} ${{ env.IMAGE_NAME }}:linux-ppc64le
+
+      - name: Push manifest to quay.io
```

Comment:
Just curious.. why did you decide to remove push-to-quay step that uses `redhat-actions/push-to-registry@v2` and instead use a podman step?

Reply:
Actually, to build images for amd64 and ppc64le I am using two dockerfiles, so I am creating the manifest list and adding their manifests in one list. 
I faced error while doing it from redhat-actions/push-to-registry@v2 action